modules/kubecf/install.sh

```diff
@@ -54,11 +53,25 @@ else
     operator_install_args+=(--set "global.operator.watchNamespace=scf")
 fi
 
+if [[ "${DOCKER_REGISTRY}" != "registry.suse.com" ]]; then
```

Comment:
Nitpicking: I would move this block after the `echo "Installing CFO"` line.

Reply:
I'm unsure about this one. The other args are all set before this is printed, and it should be the last thing that prints before actually the helm chart/release.